signature-specification.md

```diff
@@ -84,20 +84,42 @@ Notary v2 requires Payload to be the content **descriptor** of the subject manif
-  The value MUST be equal or greater than the `Creation time`
-  This is an OPTIONAL attribute.
+Notary v2 requires the signature envelope to support the following signed attributes/claims.
+
+#### Standard attributes
+
+- **Notary Signature scheme** (critical): The trust model used by Notary currently supports the following values for this claim `notary.signer.x509` and `notary.signingservice.x509`. Both signature schemes use the same set of claims defined in this document. One difference is that `notary.signer.x509` uses a TSA signature to determine authenticated signing time (timestamp), and `notary.signingservice.x509` uses the trusted signing time claim instead, with optional support for TSA signatures. Signature verification MUST consider the signature to be invalid if any other value is used.
+- **Signing time**: The time at which the signature was generated. This is a REQUIRED claim for signature scheme `notary.signer.x509`. Though this claim is signed by the signing key, it’s considered unauthenticated as a signer can modify local time and manipulate this claim. More details [here](#signing-time).
+- **Trusted signing time** (critical) : The time at which the signature was generated. This is an authenticated signing time which can be generated by a trusted time-stamper, like a signing service. This is a REQUIRED claim for signature scheme `notary.signingservice.x509` . More details [here](#signing-time).
+- **Signature Expiry** (critical): The time when a signature is considered to be expired. This is an OPTIONAL claim. [Open issue](https://github.com/notaryproject/notaryproject/issues/141) tracking if this attribute should be removed.
+- **Content type** (critical): The content type of the payload. Notary currently supports OCI descriptor of a subject manifest as the payload, supported value is `application/vnd.cncf.oras.artifact.descriptor.v1+json`, other payload types MAY be supported in future. This is a REQUIRED claim.
+- **Client identifier**: The version of a client (e.g. Notation) that produced the signature. This is an OPTIONAL claim. It uses the following format `{client}/{version}` e.g. “notation/1.0.0”. This claim in intended to be used for diagnostic and troubleshooting purposes.
 
 ### Unsigned Attributes
 
-- **Certificates**: The ordered collection of X.509 certificates with a signing certificate as the first entry.
+These attributes are considered unsigned with respect to the signing key that generates the signature. These attributes are typically signed by a third party (e.g. CA, TSA).
+
+- **Certificate Chain**: This property contains the list of X.509 certificate or certificate chain. This is a REQUIRED attribute. The certificate chain is authenticated using against a trust store as part of signature validation.
+- **TSA counter signature** : The time stamp token generated for a given signature. Only [RFC3161](ietf-rfc3161) compliant TimeStampToken are supported. This is an OPTIONAL attribute.
```

Comment:
I wonder if you should call out the work by the IETF to formulate a new standard here and though this may be the future it is not standardized at this time.

Reply:
Were you referring to the [COSE x.509](https://datatracker.ietf.org/doc/html/draft-ietf-cose-x509-08) or some standard related to TSA signature? I intended this spec to be signature format agnostic, we'll cover JWS/COSE specific mapping of these attributes in their specific doc - e.g. [Notary v2 COSE](https://github.com/notaryproject/notaryproject/blob/cose-envelope/signature-envelope-cose.md).